Documentation/ContentObjects/Fluidtemplate/Index.rst

```diff
@@ -519,11 +530,17 @@ variables
 Example
 =======
 
+..  versionadded:: 13.1
+    It is recommended to use :ref:`PAGEVIEW <cobj-pageview>` for page templates
+    starting with TYPO3 v13.1. See
+    :ref:`How to migrate to PAGEVIEW <cobj-fluidtemplate-migration>`
+
 The Fluid template in
-:file:`EXT:site_default/Resources/Private/Templates/MyTemplate.html` could look
+:file:`EXT:my_sitepackage/Resources/Private/Templates/MyTemplate.html` could look
```

Comment:
Maybe we can add a hint about the fact that the first letter has to be upper case in order to function well.

Reply:
This example here is still in the FLUIDTEMPLATE TypoScript object. I only added a note to send ppl over to the PAGEVIEW template